cross_country_aggregate_dlt.py

```diff
@@ -264,12 +264,13 @@ def expenditure_by_country_geo1_year():
 @dlt.table(name=f'expenditure_by_country_admin_func_sub_econ_sub_year')
 def expenditure_by_country_admin_func_sub_econ_sub_year():
     with_decentralized = (dlt.read('boost_gold')
+        .withColumn("is_foreign", F.when(F.col("is_foreign").isNull(), False).otherwise(F.col("is_foreign")))
         .groupBy("country_name", "year", "admin0", "admin1", "admin2", "func", "func_sub", "econ", "econ_sub", "is_foreign").agg(
-            F.sum("executed").alias("expenditure")
+            F.sum("executed").alias("expenditure"),
+            F.sum("approved").alias("approved")
```

Comment:
Can we alias "approved" to "budget" to keep in line with the language of "expenditure"?